### PR TITLE
Add error processing of FullScan exception

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (`hydra unit-tests`)~~
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3058,8 +3059,12 @@ def run_fullscan(self, ks_cf, loader_node, db_node):
...
         self.log.info('Fullscan for ks.cf: {}'.format(ks_cf))
 
-        result = loader_node.run_cqlsh(cmd='select count(*) from {}'.format(ks_cf),
-                                       timeout=3600, verbose=False, target_db_node=db_node)
+        try:
+            result = loader_node.run_cqlsh(cmd='select count(*) from {}'.format(ks_cf),
+                                           timeout=3600, verbose=False, target_db_node=db_node, connect_timeout=10)
+        except (UnexpectedExit, Failure) as details:
+            self.log.error('Fullscan command finished with errors: {}'.format(details))
+            result = details.result
```
Comment (fruch) [inline sdcm/cluster.py]: can we also send an `FullScanEvent(type='error')` to indicacted there's an error, with the details ?
i.e. we want error visible as event, and not only in the log

Comment (bentsi) [reply sdcm/cluster.py]: I think better to make `FullScanEvent(type='finish'....` event with dynamic severity, like we have for reactor stalls DB log event
@aleksbykov can you improve the finish event and make the severity dynamic?

**Comment by aleksbykov** [comment]
@bentsi did i correctly understand u, that severity should be set based on for example result exit code?
if yes, then it already dynamically set:
```
if result:
  if result['exit_code'] != 0 or result['stderr']:
    self.severity = Severity.ERROR
    self.msg += " {1.result}"
    self.publish()
```

File: sdcm/cluster.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/cluster.py]: Added to FullScanEvent code which change the type of event if exit_code of command is not 0.

**Comment by bentsi** [comment]
@aleksbykov oh, now I see that dynamic severity had been already implemented, so please disregard the comment

**Comment by bentsi** [comment]
this can't be backported since 3.1 doesn't include new run_cqlsh method

**Pull request merged**